validator/src/main.rs

```diff
@@ -1946,7 +1946,7 @@ pub fn main() {
                 .long("shred-receiver-address")
                 .value_name("SHRED_RECEIVER_ADDRESS")
                 .takes_value(true)
-                .help("Shred receiver listening address")
+                .help("Validator will forward all shreds to this address in addition to normal turbine operation. Set to empty string to disable.")
```

Comment:
nit: omit to disable?

Reply:
yeah, if they omit at startup it will be disabled, will fix help